resolwe_bio/utils/test.py

```diff
@@ -15,18 +15,19 @@
 )
 
 
-def skipDockerFailure(reason):
-    """Skip the decorated test unless ``TESTS_SKIP_DOCKER_FAILURES``
-    Django setting is set to ``False``. ``reason`` should describe why
-    the test is being skipped.
+def skipDockerFailure(reason):  # pylint: disable=invalid-name
+    """Skip the decorated tests.
+
+    Unless ``TESTS_SKIP_DOCKER_FAILURES`` Django setting is set to
+    ``False``. ``reason`` should describe why the test is being skipped.
     """
     if getattr(settings, 'TESTS_SKIP_DOCKER_FAILURES', True):
         return unittest.skip(reason)
     return lambda func: func
 
 
-def skipUnlessLargeFiles(*files):
-    """Skip the decoreted tests unless the given large files are available.
+def skipUnlessLargeFiles(*files):  # pylint: disable=invalid-name
+    r"""Skip the decoreted tests unless a large file is available.
```

Comment:
Why is a raw string needed?

Also, there can be multiple large files, so maybe it's better to write:

- `Skip the decorated tests unless specified large files are available.` or just
- `Skip the decorated tests unless large files are available.` (Please, also fix the mistake in `decoreted` -> `decorated`)